src/lib/gamepad.test.tsx

```diff
@@ -42,22 +42,20 @@ it('gamepad controls work', async () => {
   // load election from localStorage
   setElectionInLocalStorage()
   setStateInLocalStorage()
-  const { getByText } = render(<App />)
+  const { getByText, getAllByText } = render(<App />)
 
   currentCard = getNewVoterCard()
   advanceTimers()
-  await wait(() => getByText(/Precinct: Center Springfield/))
+  await wait(() => getByText(/Center Springfield/))
 
-  // for test coverage, we test pressing left and right on gamepad here, should do nothing
-  handleGamepadButtonDown('DPadLeft')
-  handleGamepadButtonDown('DPadRight')
-
-  // Go to Voting Instructions
-  fireEvent.click(getByText('Get Started'))
+  // // for test coverage, we test pressing left and right on gamepad here, should do nothing
+  // handleGamepadButtonDown('DPadLeft')
+  // handleGamepadButtonDown('DPadRight')
```

Comment:
Intentionally commented out?

Reply:
meant to delete. no longer necessary due to removal of screen.